oaknational/Oak-Web-Application
# Feat/242 fixed header

## Description

- Creates a wrapper component for other headers that is `position: fixed`
- Wrapp site header and app header in fixed header
- Make responsive
- Remove sign in for now

## Issue(s)

Fixes #242

## How to test

1. Go to https://samara-git-feat-242-fixed-header-oak-national-academy.vercel.app/
2. Scroll 
3. You should see content scrolling under the header which remains always at the top
4. View on mobile
5. Header collapses

## Screenshots

<img width="1283" alt="Screenshot 2022-07-12 at 12 49 45" src="https://user-images.githubusercontent.com/2959739/178483362-b5a16411-02fa-4f17-bb32-052e48a14201.png">


## Checklist
